kuksa_databroker/Dockerfile

```diff
@@ -27,9 +27,13 @@ FROM --platform=$TARGETPLATFORM debian:buster-slim
 
 COPY --from=builder workdir/databroker /app/databroker
 
+COPY ./data/vss-core/vss_release_3.0.json vss_release_3.0.json
+
 ENV KUKSA_DATA_BROKER_ADDR=0.0.0.0
 ENV KUKSA_DATA_BROKER_PORT=55555
 
 EXPOSE $KUKSA_DATA_BROKER_PORT
 
 ENTRYPOINT [ "/app/databroker" ]
+
+CMD ["--metadata", "vss_release_3.0.json"]
```

Comment:
Instead of adding it as an argument in the docker file, you could set a default ENV for it.

That would make it:

1. possible to add additional arguments (without adding --metadata manually again).
2. easy to remove the default by manually setting an empty KUKSA_DATA_BROKER_METADATA_FILE.

Reply:
Changed accordingly